<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reglage</class>
 <widget class="QDialog" name="reglage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
qproperty-alignment:AlignCenter;}</string>
     </property>
     <property name="text">
      <string>Mode :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>220</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
qproperty-alignment:AlignCenter;}</string>
     </property>
     <property name="text">
      <string>Son :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>87</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
qproperty-alignment:AlignCenter;

}</string>
     </property>
     <property name="text">
      <string>Langage :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="iconn.qrc">:/ICON/3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="iconn.qrc">:/ICON/2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="iconn.qrc">:/ICON/1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>120</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
 background-color: rgb(66, 117, 102);
border-radius:10px;
border-style:solid;
border-width:2px;
border-color:#9999aa;
color:white;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>Francais</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>1280</width>
       <height>720</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>437</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_play">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>320</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
 background-color: rgb(66, 117, 102);
border-radius:10px;
border-style:solid;
border-width:2px;
border-color:#9999aa;
color:white;
}
QPushButton:enabled{
 background-color: rgb(66, 117, 102);
color:white;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0.5, y1:0.0342727, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
color:#fffffe;
}

QPushButton:hover:!pressed{
 background-color:   rgb(221,222,217);
color:#0c2f70;
}

QPushButton:disabled{
background-color:rgb(66, 117, 102);
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="iconn.qrc">
       <normaloff>:/ICON/play.png</normaloff>:/ICON/play.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pause">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>320</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
 background-color: rgb(66, 117, 102);
border-radius:10px;
border-style:solid;
border-width:2px;
border-color:#9999aa;
color:white;
}
QPushButton:enabled{
 background-color: rgb(66, 117, 102);
color:white;
}

QPushButton:pressed{
 background-color: qlineargradient(spread:pad, x1:0.5, y1:0.0342727, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
color:#fffffe;
}

QPushButton:hover:!pressed{
 background-color:   rgb(221,222,217);
color:#0c2f70;
}

QPushButton:disabled{
background-color:rgb(66, 117, 102);
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="iconn.qrc">
       <normaloff>:/ICON/pause.png</normaloff>:/ICON/pause.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>320</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
qproperty-alignment:AlignCenter;}</string>
     </property>
     <property name="text">
      <string>Musique:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="iconn.qrc">:/ICON/musical-note.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>330</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius:10px;
border-style:solid;
border-width:3px;
border-color:#9999aa;}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>horizontalSlider</zorder>
    <zorder>comboBox</zorder>
    <zorder>radioButton</zorder>
    <zorder>pushButton_play</zorder>
    <zorder>pushButton_pause</zorder>
    <zorder>label_10</zorder>
    <zorder>label_9</zorder>
    <zorder>label_8</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="iconn.qrc"/>
 </resources>
 <connections/>
</ui>
